mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/model/path/spi/PojoPathFilter.java

```diff
@@ -31,4 +29,8 @@
 	 */
 	boolean test(S paths);
 
+	static <S> PojoPathFilter<S> empty() {
+		return ignored -> false;
+	}
```

Comment:
Same question about lambdas as it was moved here.

Reply:
Same answer, I fixed this one.